about.html

```diff
@@ -0,0 +1,89 @@
+          <li><a class="menu-item" href="index.html#program">Program</a></li>
+          <li><a class="menu-item" href="#logo">Logo</a></li>
+          <li><a class="menu-item" href="#past">Past Conferences</a></li>
+          <li><a class="menu-item" href="#partner">News</a></li>
+          <li><a class="nav-btn" href="#campaing">Campaign</a></li>
+        </ul>
+      </div>      
+    </nav>
+    <div class="about-item-cont">
+      <hgroup class="about-main-title">
+        <p class="about-tagline">"Hello World!"</p>
+        <h1 class="about-title">PERUVIAN SOFTWARE CONFERENCE 2023</h1>
+      </hgroup>
+      <p class="about-subheading">The Peruvian Software Conference (formerly Peruvian Science
+        and Information Conference) is a research conference held in Paracas, Perú 
+        since 2013. The annual conference series included keynotes, special sessions, 
+        poster presentations, tutorials, workshops and papers.</p>
+      <p class="email-us">Please contact us per Email for any further questions about CC Global Summit 2022!</p>
+      <a class="email-us-link" href="">ccsummit2015@cckorea.org</a>
+    </div>    
+  </header>
+  <main>
+    <section id="logo">
+      <h2 class="sec-title">Paracas<span class="orange">Soft</span><br>Peruvian Software Conference 2023 Logo</h2>
+      <p class="sec-subtitle">The logo of PS Conference 2023 was decided through the logo competition from
+        8. October to 7, November.
+      </p>
+      <div class="prog-card-cont flex-cont">
+        <img class="logo-winner" src="images/logo-psc.svg" alt="PS Conference 2023 Logo">
+      </div>      
+    </section>
+    <section id="past" class="speakers">
+      <h2 class="sec-title">See the past PS Conferences</h2>
+      <p class="sec-subtitle">Take a look at the last two PS Conferences wich took place
+        in Lima and in Arequipa.</p>
+      <div class="speak-card-cont">
+        <img class="speak-card img-shrink" src="images/ps.conf-2020.png" alt="">
+        <img class="speak-card img-shrink" src="images/ps-conf-2018.png" alt="">
+      </div>
+    </section>
+    <section id="partner" class="partner-cont">
+      <h2 class="sec-title sec-white-title">Partner</h2>
+      <div class="partner-cont flex-cont">
+        <a class="partner-logo" href=""><img src="images/mozilla.svg" alt=""></a>
+        <a class="partner-logo" href=""><img src="images/google.svg" alt=""></a>
+        <a class="partner-logo" href=""><img src="images/naver.svg" alt=""></a>
+        <a class="partner-logo" href=""><img src="images/daumkakao.svg" alt=""></a>
+        <a class="partner-logo" href=""><img src="images/airbnb.svg" alt=""></a>
+      </div>
+    </section>
+    <footer class="footer-cont flex-cont">
+      <img class="footer-logo" src="images/logo-ps.svg" alt="">
+      <div class="footer-desc">
+        <p>2023 Paracas<span class="orange">Soft</span>.</p>
+        <p>Some Rights Reserved.</p>
+      </div>      
+    </footer>
+  </main>
```

Comment:
- [ ] Please note that the footer design for the `about page` should look dark which is different for your design. Please look at the comparison below 👇 
**Given design**
![image](https://user-images.githubusercontent.com/90222110/204054025-9fef76ad-881c-400f-9394-4995b1b9c45f.png)
**Your design**
![image](https://user-images.githubusercontent.com/90222110/204054075-d32a7f1a-5563-4a94-a275-ef8b0b333ecf.png)